x/stake/keeper/validator.go

```diff
@@ -739,3 +738,18 @@ func ensureValidatorFound(found bool, ownerAddr []byte) {
 		panic(fmt.Sprintf("validator record not found for address: %X\n", ownerAddr))
 	}
 }
+
+//__________________________________________________________________________
+
+// XXX remove this code - this is should be superceded by commission work that bez is doing
+// get a single validator
+func (k Keeper) UpdateValidatorCommission(ctx sdk.Context, addr sdk.ValAddress, newCommission sdk.Dec) sdk.Error {
```

Comment:
This will be removed before this PR is merged - just need to work from #2365 to update properly